console/nginx_conf.go

```diff
@@ -22,7 +22,7 @@ var NginxConf = `
 
 worker_processes auto;
 error_log /var/log/nginx/error.log;
-pid /run/nginx.pid;
+pid /var/lib/nginx/tmp/nginx.pid;
```

Comment:
```suggestion
pid /var/log/nginx/nginx.pid;
```
what if we add it to the same location as error.log ?
that way we don;t have to provide write permission for 2 separate paths...

Reply:
okay... if it needs access to this path anyway then we are good...